sdk/ml/azure-ai-ml/tests/conftest.py

```diff
@@ -613,7 +613,6 @@ def credentialless_datastore(client: MLClient, storage_account_name: str) -> Azu
 @pytest.fixture()
 def enable_pipeline_private_preview_features(mocker: MockFixture):
     mocker.patch("azure.ai.ml.entities._job.pipeline.pipeline_job.is_private_preview_enabled", return_value=True)
-    mocker.patch("azure.ai.ml.dsl._pipeline_component_builder.is_private_preview_enabled", return_value=True)
     mocker.patch("azure.ai.ml._schema.pipeline.pipeline_component.is_private_preview_enabled", return_value=True)
```

Comment:
shall we also remove this schema check?

Reply:
it's used for control flow node checks, I think we should keep them. 
![image](https://user-images.githubusercontent.com/24237253/209070507-0617e8e2-e848-408f-90c5-93c536b0b0f3.png)